patterns/events-three-columns-images-and-titles.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Title: Events three columns with event images and titles
+ * Slug: twentytwentyfive/events-three-columns-images-and-titles
+ * Categories: featured, query
+ * Description: A header with title and text and three columns that show 3 events with their images and titles.
+ * Keywords: events, columns, images
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:group {"metadata":{"categories":["featured"],"patternName":"twentytwentyfive/events-three-columns-images-and-titles","name":"Events three columns with event images and titles"},"align":"wide","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"},"blockGap":"0"}},"layout":{"type":"constrained"}} -->
```

Comment:
Please remove the metadata.

Reply:
The outer group block needs to be full width, otherwise there is no left and right spacing when a background color is added.
Please set the top and bottom margin to 0, so that there is no extra spacing between full width sections:

![image](https://github.com/user-attachments/assets/9e04d65c-8e72-4e87-88a5-23f4cf85c160)